src/site/content/en/fast/efficiently-load-third-party-javascript/index.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: post
+title: Efficiently load third-party JavaScript
+subhead: Avoid common pitfalls of using third-party scripts and improve load times and user experience.
+authors:
+  - mihajlija
+date: 2019-07-29
+description: |
+  Learn how to avoid common pitfalls of using third-party scripts and improve load times and user experience.
+hero: hero.jpg
+alt: Aerial view of shipping containers.  
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - performance
+  - third-party
+  - javascript
+---
+
+If a third-party script is [slowing down](/third-party-javascript/) your page load, you have two options to improve performance:
+
+* Remove it if it doesn't add clear value to your site.
+
+* Optimize the loading process.
+
+This guide explains how to optimize the loading process of third-party scripts with the following techniques: 
+
+1. Using the `async` or `defer` attribute on `<script>` tags
+
+2. Establishing early connections to required origins
+
+3. Lazy-loading
+
+4. Optimizing how you serve third-party scripts 
+
+## Use `async` or `defer`
+
+Because [synchronous scripts](link-to-intro) delay DOM construction and rendering, you should always load third-party scripts asynchronously unless the script has to run before the page can be rendered.
+
+The `async` and `defer` attributes tell the browser that it may go on parsing the HTML while loading the script in the background, and then execute the script after it loads. This way, script downloads don’t block DOM construction and page rendering. The result is that the user can see the page before all scripts have finished loading.
```

Comment:
```suggestion
The `async` and `defer` attributes tell the browser that it may go on parsing the HTML while loading the script in the background, and then execute the script after it loads. This way, script downloads don't block DOM construction and page rendering. The result is that the user can see the page before all scripts have finished loading.
```
Curly apostrophe to straight.